lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1937,6 +1897,62 @@ export class UserAgentApplication {
     }
     return response;
   }
+
+  /**
+   * This method must be called for processing the response received from AAD. It extracts the hash, processes the token or error, saves it in the cache and calls the registered callbacks with the result.
+   * @param {string} authority authority received in the redirect response from AAD.
+   * @param {TokenResponse} requestInfo an object created from the redirect response from AAD comprising of the keys - parameters, requestType, stateMatch, stateResponse and valid.
+   * @param {User} user user object for which scopes are consented for. The default user is the logged in user.
+   * @param {ClientInfo} clientInfo clientInfo received as part of the response comprising of fields uid and utid.
+   * @param {IdToken} idToken idToken received as part of the response.
+   * @ignore
+   * @private
+   * @hidden
+   */
+  private saveAccessToken(response: AuthResponse, authority: string, parameters: any, clientInfo: string): AuthResponse {
```

Comment:
location change of the method? or something changed here. Github diff is not helping me.

Reply:
Yes, no other changes.

Reply:
moved this back to its original location for the purposes of this PR